agent/composer-dependency-analyser.php

```diff
@@ -3,4 +3,4 @@
 use ShipMonk\ComposerDependencyAnalyser\Config\Configuration;
 
 return (new Configuration)
-    ->ignoreUnknownFunctions(['signature']);
+    ->ignoreUnknownFunctions(['signature', 'run']);
```

Comment:
We'll be able to remove this completely once we migrate off Pest.

Needed as part of the migration.